src/particles/ReferenceParticle.H

```diff
@@ -160,6 +160,23 @@ namespace impactx
             return *this;
         }
 
+        /** Get reference particle magnetic rigidity
+         *
+         * @returns magnetic rigidity Brho in T-m
+         */
+        AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
+        amrex::ParticleReal
+        rigidity_Tm () const
```

Comment:
Let's add this also in Python at this location:
https://github.com/ECP-WarpX/impactx/blob/aaf11831f4223a7df57bfef372656bed14d07e14/src/python/ReferenceParticle.cpp#L34-L40

Reply:
Added a commit :)